server/src/main/java/io/deephaven/server/session/SessionState.java

```diff
@@ -708,15 +708,14 @@ public T get() {
             if (session != null && session.isExpired()) {
                 throw Exceptions.statusRuntimeException(Code.UNAUTHENTICATED, "session has expired");
             }
-
+            final T localResult = result;
```

Comment:
tweaked for debugging, or does this serve a purpose?

Reply:
`result` is volatile, so we should only do the volatile read once in this method for correctness (also, performance, but I doubt that matters here).